updater/lib/dependabot/file_fetcher_command.rb

```diff
@@ -22,12 +22,9 @@ def perform_job
         raise "base commit SHA not found" unless @base_commit_sha
 
         # We don't set this flag in GHES because there's no point in recording versions since we can't access that data.
-        # TODO: The flag is named `record_ecosystem_versions` because `package_manager_version` is getting renamed to
-        # to that shortly... but splitting into separate PR's for lower risk/easiery testability. Once the follow-on PR
-        # with the rename lands, the name confusion will disappear.
         if Experiments.enabled?(:record_ecosystem_versions)
-          version = file_fetcher.package_manager_version
-          api_client.record_package_manager_version(version[:ecosystem], version[:package_managers]) unless version.nil?
+          e_versions = file_fetcher.ecosystem_versions
+          api_client.record_ecosystem_versions(e_versions) unless e_versions.nil?
```

Comment:
```suggestion
ecosystem_versions = file_fetcher.ecosystem_versions
api_client.record_ecosystem_versions(ecosystem_versions) unless ecosystem_versions.nil?
```

Sorry, it's really a really minor but deeply engrained nitpick 😅

Reply:
hah, I had that originally then decided it was too long. But I didn't want `versions` like it used to be as that felt too ambiguous. 👍 for this from me.